app/src/main/AndroidManifest.xml

```diff
@@ -57,7 +57,7 @@
             android:theme="@style/AppTheme" />
 
         <service
-            android:name=".service.ListenScrobbleService"
+            android:name=".service.ListeningService"
```

Comment:
This naming is wrong because it sounds very generic and is confusing. Something like `ListenSubmissionService` is better

Reply:
Sounds good. Will do.